Added slack channel link in Readme

coderfender opened the pull request:
Added slack channel URL in the docs so that users can take advantage of it and ask any questions regarding the setup , development and possibly contribute.

Agirish commented on README.md:
```diff
@@ -8,22 +8,23 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 
-Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md).
+Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md) and 
+also use slack channel for discussions.
```
I'll suggest skipping this change

cgivre replied:
@Agirish What's the objection to publicizing the Slack channel?

Agirish commented on README.md:
```diff
@@ -8,22 +8,23 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 Please say hello on the [Apache Drill mailing list](http://drill.apache.org/mailinglists/)
-or join our [Google Hangouts](http://drill.apache.org/community-resources/) for more information.
-(More information can be found at the [Apache Drill website](http://drill.apache.org/)).
+or join our [Slack Channel](https://apache-drill.slack.com/) if you need help with using or developing Apache Drill.
```
We have a public invite link for Slack which needs to be included as well. I'll look that up

cgivre replied:
Maybe we could create a bit.ly link?

Agirish replied:
Link: https://bit.ly/2VM0XS8

Agirish replied:
So we could update to say, the Slack channel is https://apache-drill.slack.com, and to join, use the bit.ly link.

coderfender commented:
@Agirish  , @cgivre  I fixed the readme as per the comments mentioned above. Please review and let me know if you think something else needs to be changed.

vdiravka commented on README.md:
```diff
@@ -8,22 +8,22 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 ## Developers
 
-Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md).
+Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md)
```
It could be useful to add the link to this wiki for devs:
https://github.com/paul-rogers/drill/wiki
cc: @paul-rogers

vdiravka commented on README.md:
```diff
@@ -8,22 +8,22 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 Please say hello on the [Apache Drill mailing list](http://drill.apache.org/mailinglists/)
-or join our [Google Hangouts](http://drill.apache.org/community-resources/) for more information.
-(More information can be found at the [Apache Drill website](http://drill.apache.org/)).
+or [join](https://bit.ly/2VM0XS8) our [Slack Channel](https://apache-drill.slack.com/) if you need help with using or developing Apache Drill.
```
What about proper link to Google Hangouts with a schedule?
http://meet.google.com/yki-iqdf-tai

vdiravka replied:
And `http://drill.apache.org/community-resources` link still can be useful in `Join the community` paragraph

coderfender replied:
I am not aware about the Google Hangouts schedule and it would be create if you could provide some context about this. Also , I made changes as requested .

vdiravka replied:
You can leave the info about Google Hangouts, since it is still actual and add the new info about Slack Channel

vdiravka commented:
@coderfender Sorry to see the PR is closed. Do you still want to merge the changes?

coderfender commented:
@vdiravka  , @Agirish , @cgivre , I renamed this branch and somehow this pull request got closed ( may be I didnt do it properly) . Can you guys please review the other PR ([link](https://github.com/apache/drill/pull/1793)) where in I fixed all the issues you have mentioned?
